llvm/lib/CAS/Utils.cpp

```diff
@@ -84,3 +84,34 @@ Error cas::walkFileTreeRecursively(
 
   return Error::success();
 }
+
+Error cas::walkObjectsRecursively(
+    CASDB &CAS, CASID ID,
+    function_ref<Error(const std::pair<CASID, unsigned> &)> Callback) {
+
+  BumpPtrAllocator Alloc;
+  StringSaver Saver(Alloc);
+  SmallString<128> PathStorage;
```

Comment:
Is this unused variable?

Reply:
Yes it is, I forgot to remove it, sorry about that